includes/definitions/discovery/ciscosb.yaml

```diff
@@ -71,8 +71,29 @@ modules:
                     index: '{{ $index }}'
                     descr: 'FAN {{ $index }} Temperature'
                     user_func: fahrenheit_to_celsius
+                    high_limit: 50 # true for all switches
+                    low_limit: -5 # true for nearly all switches except CBS250-8T-D and CBS250-8PP-D, they would like 0°C
```

Comment:
The default setting should be inclusive for all models of this equipment.

Reply:
thank you for your feedback, I've changed it.